processing/src/main/java/io/druid/segment/IntListUtils.java

```diff
@@ -29,6 +29,8 @@ private IntListUtils() {}
 
   public static IntList fromTo(int from, int to)
   {
+    // TODO: check `from` is always smaller than or equal to `to`.
```

Comment:
I don't think it's a good idea to skip the check because of a single use case. If `to` could be less than `from` on `BoundFilter` side, we can add a check on that side and return `IntLists.EMPTY_LIST`.

Also, in Druid we try to avoid adding TODOs, add issues instead, even for relatively small things. And too small things should probably be fixed in-place. See https://github.com/metamx/java-util/pull/50/files#r77869324